rectools/metrics/classification.py

```diff
@@ -371,6 +371,26 @@ def _calc_per_user_from_confusion_df(self, confusion_df: pd.DataFrame, catalog:
         return mcc
 
 
+@attr.s
+class HitRate(SimpleClassificationMetric):
+    """
+    HitRate calculates the fraction of users for which the correct answer is included in the recommendation list
```

Comment:
There should be a dot at the end of the sentence.

Reply:
Fix type here